src/mango/src/mango_cursor_view.erl

```diff
@@ -28,6 +29,8 @@
 
 -include_lib("couch/include/couch_db.hrl").
 -include_lib("couch_mrview/include/couch_mrview.hrl").
+-include_lib("fabric/include/fabric.hrl").
```

Comment:
I'm missing why we need to include fabric.hrl.

Reply:
We need it because it has the view_row record https://github.com/apache/couchdb/blob/master/src/fabric/include/fabric.hrl#L43